grafana-plugin/src/plugin.json

```diff
@@ -620,8 +620,7 @@
     }
   ],
   "dependencies": {
-    "grafanaDependency": ">=8.3.2",
-    "grafanaVersion": "8.3",
```

Comment:
`dependencies .grafanaVersion` is deprecated according to the [docs](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#dependencies)